ern-core/src/MiniApp.js

```diff
@@ -267,14 +271,14 @@ Are you sure this is a MiniApp ?`)
         await spin(`${versionLessDependency.toString()} is not declared in the manifest. Performing additional checks.`,
                     yarn.add(DependencyPath.fromString(dependency.toString())))
 
-        const nativeDependencies = findNativeDependencies(path.join(tmpPath, 'node_modules'))
-        if (nativeDependencies.length === 0) {
+        const nativeDependencies = await nativeDependenciesLookup.findNativeDependencies(path.join(tmpPath, 'node_modules'))
+        if (nativeDependencies.all.length === 0) {
```

Comment:
nativeDependencies && nativeDependencies.all.length

Reply:
or _isEmpty()

Reply:
yes it's never going to be null/undefined, empty array by default, but will use `_.isEmpty` instead, as it is more proper